.gitignore

```diff
@@ -27,3 +27,20 @@ setup.log
 
 # Local OPAM switch
 _opam/
+
+.DS_Store
+.DS_Store
+bounce/.DS_Store
```

Comment:
You don't need to add each directory. It should be enough just to have .DS_Store in the file once.